snuba/datasets/configuration/transactions/storages/transactions.yaml

```diff
@@ -157,6 +157,7 @@ schema:
       },
       { name: app_start_type, type: String },
       { name: profile_id, type: UUID, args: { schema_modifiers: [nullable] } },
+      { name: replay_id, type: UUID, args: { schema_modifiers: [nullable] } },
```

Comment:
why is this needed for transactions?

Reply:
do you also need to add a replay_id column like this to discover.yaml?

Reply:
because we want to query on this column which exists in the transactions dataset